scripts/lib/clean_yarn_cache.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import sys
+
+ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.append(ZULIP_PATH)
+from datetime import date, datetime
+
+from dateutil.relativedelta import relativedelta
+
+from scripts.lib.zulip_tools import may_be_perform_purging, parse_cache_script_args, run_as_root
+
+YARN_CACHE_PATH = "../.cache/yarn/"
```

Comment:
Declaring this relative to Zulip path would be better

Reply:
Yeah, we should avoid anything that depends on CWD when writing code under `scripts/`.